tests/test_data/collection_exercise/collection_exercise_details.json

```diff
@@ -30,5 +30,19 @@
     "longName": "Business Register and Employment Survey",
     "shortName": "BRES",
     "surveyRef": "221"
-  }
+  },
+  "collection_instruments": [
+    {
+      "classifiers": {
+        "COLLECTION_EXERCISE": [
+          "e33daf0e-6a27-40cd-98dc-c6231f50e84a"
+        ],
+        "RU_REF": [],
+        "SURVEY_ID": "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87"
+      },
+      "file_name": "collection_instrument.xlsx",
+      "id": "f732afbe-c710-4c95-a8d3-6644833195a7",
+      "surveyId": "cb0711c3-0ac8-41d3-ae0e-567e5ea1ef87"
+    }
+  ]
```

Comment:
No new line at end of file.

Reply:
Don't think it's necessary in this case as it's json and it's only pep8 thing